source/kernel/src/drivers/mod.rs

```diff
@@ -17,5 +17,6 @@
 //!       the [serial_mux] module.
 
 pub mod emb_display;
+pub mod forth_spawnulator;
```

Comment:
nit: does this really belong in `drivers`? maybe we need a `mod services` or something?

Reply:
Yeah... I dunno what the naming/hierarchy for this should be. It probably should be services.

Reply:
not a big deal, just thought i'd mention it